CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+3.9.0
+------
+
+**ENHANCEMENTS**
+- Add support for installing Intel OneAPI Base Toolkit and HPC Toolkit, and Intel Python.
```

Comment:
Let's specify the version of the installed packages.

Reply:
Done